app/models/labware.rb

```diff
@@ -72,14 +72,12 @@ def receptacle_proxy
   belongs_to :purpose, foreign_key: :plate_purpose_id, optional: true, inverse_of: :labware
 
   has_one :spiked_in_buffer_links,
-          # If we try to use the Rails 6 version, Rails 5 doesn't seem to perform the join
```

Comment:
Comments left over from when I had different Rails5/Rails6 implementations, but managed to fix this.